clients/v1_shell/table_model.go

```diff
@@ -0,0 +1,150 @@
+package v1shell
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/evertras/bubble-table/table"
+	"github.com/fatih/color"
+	"github.com/influxdata/influx-cli/v2/api"
+	"github.com/mattn/go-isatty"
+)
+
+const (
+	colPadding int = 2
+)
+
+type Model struct {
+	rows        []table.Row
+	allCols     []table.Column
+	colNames    []string
+	simpleTable table.Model
+	totalWidth  int
+}
+
+func NewModel(res api.InfluxqlJsonResponseSeries) Model {
```

Comment:
How does the `>` at the end of a wide table work? I can't see where that gets set up.

Reply:
Happens in the bubble-table library, it's a new feature upstream that I suggested.